packages/validators/src/schemas/componentDefinition.ts

```diff
@@ -22,14 +22,120 @@ export const ComponentDefinitionSchema = z.object({
   id: DefinitionPropertyKeySchema,
   variables: z.record(
     DefinitionPropertyKeySchema,
-    z.object({
-      // TODO - extend with definition of validations and defaultValue
-      displayName: z.string().optional(),
-      type: DefinitionPropertyTypeSchema,
-      description: z.string().optional(),
-      group: z.string().optional(),
-    }),
+    z
+      .object({
+        // TODO - extend with definition of validations and defaultValue
+        displayName: z.string().optional(),
+        type: DefinitionPropertyTypeSchema,
+        description: z.string().optional(),
+        group: z.string().optional(),
+        defaultValue: z.union([z.string(), z.number(), z.undefined(), z.any()]).optional(),
```

Comment:
The `PrimitiveValueSchema` type in `src/schemas/v2023_09_28/experience.ts` might be a good option here.
```suggestion
        defaultValue: PrimitiveValueSchema.optional(),
```

Reply:
ohh, thanks. I put that in to get it to work and mean to come back to it.